tools/aquainfra_importer/aquainfra_importer.xml

```diff
@@ -0,0 +1,13 @@
+<tool id="aquainfra_importer" name="AquaINFRA Importer" tool_type="data_source" version="1.0" profile="22.05">
```

Comment:
```suggestion
<tool id="aquainfra_importer" name="AquaINFRA Importer" tool_type="data_source" version="1.0" profile="20.09">
```

Higher profile versions will need changes to the default `data_source.py`. I can show you what these are, but this is the quick solution.